src/components/Tool/GoogleImport/GoogleImport.tsx

```diff
@@ -0,0 +1,553 @@
+                            {!!account?.contactGroups.length && (
+                              <Box
+                                sx={{
+                                  display:
+                                    importAll === 'false' ? 'block' : 'none',
+                                }}
+                              >
+                                <BorderBox>
+                                  <HeaderBox>
+                                    <Grid
+                                      container
+                                      sx={{ alignItems: 'center' }}
+                                    >
+                                      <Grid item xs={4}>
+                                        <BoldTypography>
+                                          {t('Contact Group')}
+                                        </BoldTypography>
+                                      </Grid>
+                                      <Grid item xs={4}>
+                                        <BoldTypography>
+                                          {t('Tags for Group')}
+                                        </BoldTypography>
+                                      </Grid>
+                                      <Grid
+                                        item
+                                        xs={4}
+                                        sx={{ textAlign: 'right' }}
+                                      >
+                                        <ButtonGroup size="small">
+                                          <Button
+                                            onClick={() =>
+                                              setFieldValue(
+                                                'groups',
+                                                account?.contactGroups.map(
+                                                  (group) => group?.id,
+                                                ),
+                                              )
+                                            }
+                                          >
+                                            {t('Check All')}
+                                          </Button>
+                                          <Button
+                                            onClick={() =>
+                                              setFieldValue('groups', [])
+                                            }
+                                          >
+                                            {t('Uncheck All')}
+                                          </Button>
+                                        </ButtonGroup>
+                                      </Grid>
+                                    </Grid>
+                                  </HeaderBox>
+                                  <Box
+                                    sx={{
+                                      marginX: theme.spacing(2),
+                                    }}
```

Comment:
Did you know that at least for `Box`, I believe you can write this as `<Box mx={2}>`?

Reply:
Ah I forgot about that